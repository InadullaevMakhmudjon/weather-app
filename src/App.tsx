import React, { FunctionComponent } from 'react'

interface IProps {
    name: string
}

const App: FunctionComponent<IProps> = ({ name }) => (
    <h1 className="text-red-500">Hey {name}</h1>
)

export default App