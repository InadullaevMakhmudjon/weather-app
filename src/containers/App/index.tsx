import React, { FunctionComponent } from 'react'
import BackgroundImage from '../../assets/cloudy.jpg'
import MainPanel from '../../views/MainPanel'
import InfoPanel from '../../views/InfoPanel'

interface IProps {}

const App: FunctionComponent<IProps> = () => {
    return (
        <div className="invisible sm:visible w-[100vw] h-[100vh] fixed grid grid-cols-custom overflow-auto">
            <img className="w-full h-full absolute" alt="loading..." src={BackgroundImage}/>
            <MainPanel />
            <InfoPanel />
        </div>
    )
}

export default App