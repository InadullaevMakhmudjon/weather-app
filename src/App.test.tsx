import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

test('App test created', () => {
  const component = renderer.create(<App name="Hello" />).toJSON();
  expect(component).toMatchSnapshot()
});