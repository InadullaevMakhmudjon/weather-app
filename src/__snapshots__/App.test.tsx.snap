// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test created 1`] = `
<h1
  className="text-red-500"
>
  Hey 
  Hello
</h1>
`;
